lib/python/dataUtils.py

```diff
@@ -6,17 +6,18 @@
 ### reading data in csv files
 
 import csv
-import numpy as np
 from datetime import datetime, timedelta
-from netCDF4 import Dataset
-from scipy.io import loadmat
 
 # use Agg backend for so code can run in background
 import matplotlib
+import numpy as np
+from matplotlib.pyplot import close, figure
+from matplotlib.ticker import ScalarFormatter
+from netCDF4 import Dataset
+from scipy.io import loadmat
+
 if matplotlib.get_backend() <> 'Agg':
     matplotlib.use('Agg')
-from matplotlib.pyplot import figure, close
-from matplotlib.ticker import ScalarFormatter
```

Comment:
This needs to be another exception. `matplotlib.use()` needs to be called before importing `pyplot`  (see [matplotlib docs](http://matplotlib.org/faq/howto_faq.html#generate-images-without-having-a-window-appear)).

However, as 'pyplot' is only used in the `plotRecent()` function below, we could just move these last two imports in there. In fact it looks like this function is not used anywhere, so we could just remove the whole function and all matplotlib imports here.